stackdriver/queue_manager_test.go

```diff
@@ -295,6 +297,44 @@ func TestSampleDeliveryTimeout(t *testing.T) {
 	c.waitForExpectedSamples(t)
 }
 
+func TestDeliveryWithMetricPrefix(t *testing.T) {
+	// Let's create an even number of send batches so we don't run into the
+	// batch timeout case.
+	n := 100
+
+	samples := make([]sample, 0, n)
+	for i := 0; i < n; i++ {
+		name := fmt.Sprintf("test_metric_%d", i)
+		samples = append(samples, sample{
+			Name: name,
+			Labels: map[string]string{
+				model.InstanceLabel: strconv.Itoa(i),
+			},
+			Value:          float64(i),
+			ResetTimestamp: 1234567890000,
+			Timestamp:      2234567890000,
+		})
+	}
+
+	c := NewTestStorageClient(t)
+	c.prefixLen = 4
```

Comment:
How is this used? I think I'm missing something.

Reply:
Sorry I missed to use the variable. Updated and the test is passing now.